### DISCUSSION TITLE
status of librc

### DISCUSSION THREAD

**Discussion by williamh** [OpenRC/openrc#440] [Ideas]
Hey all,

does anyone know of any external projects that link to librc?

If not, I am considering making the code private to OpenRC.

Thanks,

William

**Comment by vapier** [comment]
doesn't sound like anyone cares, so sounds like you can do it.  just bump the major version and wait for feedback.

shared lib is still nice for avoid code bloat even if it's an internal shared lib.

**Comment by jirutka** [comment]
> does anyone know of any external projects that link to librc?

Yes, [collectd-openrc](https://github.com/jirutka/collectd-openrc/).

> If not, I am considering making the code private to OpenRC.

Why do you wanna do that?

**Comment by navi-desu** [comment]
we'll be refactoring librc's api in the future, but not dropping it. there's a few tools that make use of it, and i'll even attempt a patch for dbus that does openrc-based activation using it.
